front/app/containers/Admin/settings/index.tsx

```diff
@@ -45,11 +45,6 @@ class SettingsPage extends React.PureComponent<
           label: formatMessage(messages.tabCustomize),
           url: '/admin/settings/customize',
         },
-        {
-          name: 'pages',
```

Comment:
We should definitely communicate this clearly to GovSuccess. Could you make a ticket in the epic so we won't forget?